Led.cpp

```diff
@@ -1,7 +1,33 @@
 #include "Led.h"
 
-Led::Led() {
+Led::Led() : pin(pin) {
 }
 
-Led::~Led() {
-}
\ No newline at end of file
+Led::Led(unsigned pin) : pin(pin) {
+    //	pinMode(pin, OUTPUT);
+}
+
+void Led::setBrightness(unsigned percent) {
+    //	analogWrite(led, precent);
+    brightness = percent;
+}
+
+void Led::turnOn() {
+    //	digitalWrite(pin, HIGH);
+}
+
+void Led::turnOff() {
+    //	digitalWrite(pin, LOW);
+}
+
+void Led::blink(unsigned n, unsigned delay1, unsigned delay2) {
```

Comment:
Another requirement for the blink function was that if the led was already on from the user calling `turnOn` then the a inverse blink would occur. Meaning it would turnoff for a short delay and then back on. REfer to the issue #46